Description: Revert back fribidi_log2vis_get_embedding_levels function
 It seems to be removed by mistake by upstream, since its documentation is 
 still there
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@users.sourceforge.net>
Bug-Debian: https://bugs.debian.org/947081
Bug: https://github.com/fribidi/fribidi/issues/134
Index: fribidi/lib/fribidi-deprecated.c
===================================================================
--- fribidi.orig/lib/fribidi-deprecated.c	2019-12-21 02:33:36.731869511 +0100
+++ fribidi/lib/fribidi-deprecated.c	2019-12-21 02:44:44.262537613 +0100
@@ -76,6 +76,19 @@
 
 
 
+FRIBIDI_ENTRY FriBidiLevel
+fribidi_log2vis_get_embedding_levels (
+  const FriBidiCharType *bidi_types,	/* input list of bidi types as returned by
+					   fribidi_get_bidi_types() */
+  const FriBidiStrIndex len,	/* input string length of the paragraph */
+  FriBidiParType *pbase_dir,	/* requested and resolved paragraph
+				 * base direction */
+  FriBidiLevel *embedding_levels	/* output list of embedding levels */
+)
+{
+  return fribidi_get_par_embedding_levels_ex (bidi_types, NULL, len, pbase_dir, embedding_levels);
+}
+
 FRIBIDI_ENTRY FriBidiCharType
 fribidi_get_type (
   FriBidiChar ch		/* input character */
